### ISSUE TITLE
Write csv without converting doubles to scientific notation

### ISSUE THREAD

**Issue by StevenWillyS** [jtablesaw/tablesaw#892]
Hello, Thanks for the great library

want to ask some question, 
regarding the column type is it possible to change the type once it's created? like i get double type from summarize(count) but all the value actually just [10.0, 20.0, 15.0], is it possible to change it to just integer?

Also is it possible to disable the scientific notation? trying to use the write csv and some big double value automatically converted to scientific notation
33996600.0 -> 3.39966E7

Thanks!

**Comment by lwhite1** [comment]
All math ops return doubles as it’s the only sane way to do it in Java.
You can convert the result to an int column using round() or possibly
asIntColumn().

I don’t recall exactly.

On avoiding scientific  notation, I don’t think so, but I consider this a
bug.

**Comment by StevenWillyS** [comment]
Oh, i see. 
Thanks for the response!

**Comment by lwhite1** [comment]
This should be fixed when PR #914 is merged. After that, you could handle this case as shown below: 

```
    Table table = Table.create("", DoubleColumn.create("doubles"));
    table.doubleColumn("doubles").append(32.32342489123).append(0.1192342224).append(1001.0);


// Create a DecimalFormat specifying a leading 0 for values between 0 and 1, and up to 11 places after the decimal
    
    DecimalFormat df = new DecimalFormat("0.#");
    df.setMaximumFractionDigits(11);

// create a NumberColumnFormatter wrapping the DF, and set it on the column 

    NumberColumnFormatter formatter = new NumberColumnFormatter(df);
    table.doubleColumn("doubles").setPrintFormatter(formatter);
     
// use the WriteOption usePrintFormatters() 
    table.write().usingOptions(CsvWriteOptions.builder("my.csv").usePrintFormatters(true).build());

```
This should produce the output values   
```
    32.32342489123
      0.1192342224 
    1001
```

**Comment by lwhite1** [comment]
merged.
